### PR TITLE
Update fr web translation for 3.8.2

### LINKED ISSUE #690: Update fr web translation for 3.8.2
Hi @jujudusud, @trebmuh,

We are getting ready for the 3.8.2 release and have created the [next-release](https://github.com/jamulussoftware/jamuluswebsite/tree/next-release) branch ([full diff](https://github.com/jamulussoftware/jamuluswebsite/compare/release..next-release)).

We would be happy if you updated the translations for **fr** by **2022-02-13**.

Please

- Start your work in your fork on a branch based on jamuluswebsite's `next-release` branch.
- Update the language-specific files using your favorite editor (or directly on Github),
- New/changed images are listed at the end of this issue. [Generate new URLs](https://github.com/jamulussoftware/jamuluswebsite/tree/release#adding-screenshots) for your image `.inc` files.
- Commit and push your changes to your fork,
- Open a Pull Request with your translations to the **next-release** branch with the subject `Update fr web translation for 3.8.2`,
- Link your PR(s) to this issue by including `Fixes #690` in the PR content.

This Issue is assigned to multiple people. Please coordinate who will translate what part.

Suggested commit and PR message:
```
Update fr web translation for 3.8.2

Fixes #690
```

Feel free to use this Issue to discuss anything you need prior to making any PR (and in your native language if needed).

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/README.md#translating).

Thanks for contributing to Jamulus!

---

**New/Changed screenshots**:

Please [replace with new URLs](https://github.com/jamulussoftware/jamuluswebsite/tree/release#adding-screenshots) for your screenshots here, and use those URLs in
your `.inc` files.

settings-profile

### PR DESCRIPTION
**Pull request by jujudusud**
# Changes

Update fr web translation for 3.8.2

## Related issues

Fixes #690

### CODE REVIEW

**Comment by jujudusud** [comment]
Work in progress:

- [x] Waiting for FR app translations,
- [x] create the FR screenshots with the FR translations activated,
- [x] push the FR screenshots into "githubusers",
- [x] update the urls in *inc files.

File: _translator-files/po/fr/Tips-Tricks-More.po [hunk 1]
```diff
@@ -247,23 +244,21 @@ msgstr "Ici, Jamulus écoute sur le canal MIDI 1. Les numéros de CC des faders
...
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:94
 #, no-wrap
 msgid "*Note*: Jamulus does not provide feedback on the state of the Solo and Mute buttons, meaning that your controller must keep track and toggle LEDs (if any) to 'on' or 'off' itself.\n"
-msgstr "*Note*: Jamulus ne fournit pas de retour sur l'état des boutons Solo et Mute, ce qui signifie que votre contrôleur doit garder la trace et basculer les LED (le cas échéant) sur \"on\" ou \"off\" lui-même.\n"
+msgstr "*Note*: Jamulus ne fournit pas de retour sur l'état des boutons Solo et slencé, ce qui signifie que votre contrôleur doit garder la trace et basculer les diode lumineuses (le cas échéant) sur \"on\" ou \"off\" lui-même.\n"
```
Comment (trebmuh) [inline _translator-files/po/fr/Tips-Tricks-More.po]: Manque un "i".

Comment (jujudusud) [reply _translator-files/po/fr/Tips-Tricks-More.po]: Merci @trebmuh ! Je m'en occupe.

Comment (jujudusud) [reply _translator-files/po/fr/Tips-Tricks-More.po]: ok with b1cf886

**Review by gilgongo — APPROVED** [review]
Looks good to me!

**Pull request merged**